### PR TITLE
Coursera initial commit - changes.

### PR DESCRIPTION
**Pull request by tguils**
Hey Dean Can you review my changes.

### CODE REVIEW

**Comment by deanmiranda** [comment]
Sure thing boss mang... your code looks amazing!

What an amazing developer you are! WOWzer

File: site/index.html [hunk 1]
```diff
@@ -1,10 +1,55 @@
...
+      <li>second item</li>
+  </ol>  
+list  
+<ul>
+      <li>one</li>
+      <li>two</li>
+  </ul>
+</div>
+<p>
+    &quote;dijasd;fja;dsklfj fad;skfj;adslkfjadskfj dkfj;adkljf;asdkfj ad hello my name&nbsp; is&nbsp; tess&quot;
```
Comment (deanmiranda) [inline site/index.html]: Did you mean to sneeze while typing this?

**Review by deanmiranda — CHANGES_REQUESTED** [review]

File: site/index.html (hunk 1 above)
Comment (tguils) [reply site/index.html]: yes

File: site/index.html [hunk 2]
```diff
@@ -23,33 +23,8 @@ <h1> Hello Coursera! </h1>
   </ul>
 </div>
 <p>
-    &quote;dijasd;fja;dsklfj fad;skfj;adslkfjadskfj dkfj;adkljf;asdkfj ad hello my name&nbsp; is&nbsp; tess&quot; 
+    hi my name is tess
```
Comment (deanmiranda) [inline site/index.html:26]: Great JORB!

**Pull request merged**